app/views/namespaces/components/_table_row.html.slim

```diff
@@ -0,0 +1,16 @@
+tr :class="scopeClass"
+  td
+    a{ :href="namespace.links.self.href" }
```

Comment:
If `:class ="scopeClass"` works, then `:href="..."` should also work. Will try without the curly brackets.